report/src/main/kotlin/com/hartwig/actin/report/pdf/chapters/MolecularDetailsChapter.kt

```diff
@@ -62,12 +63,23 @@ class MolecularDetailsChapter(
             )
             if (molecular.hasSufficientQualityButLowPurity()) {
                 val purityString = molecular.characteristics.purity?.let { Formats.percentage(it) } ?: "NA"
-                orangeMolecularTable.addCell(
-                    Cells.createContentNoBorder(
-                        ("Low tumor purity (${purityString}) indicating that potential (subclonal) " +
-                                "DNA aberrations might not have been detected & predicted tumor origin results may be less reliable")
+                val cuppaMode = molecular.characteristics.predictedTumorOrigin?.cuppaMode()
+                if (cuppaMode == CuppaMode.WGTS) {
```

Comment:
Unnecessary duplication here. You can interpolate an if statement within the message string which can optionally add the (WGTS) and eliminate the copy/paste of the whole message.